README.md

```diff
@@ -19,8 +19,22 @@ This Cabal Git repository contains the following main packages:
 The canonical upstream repository is located at
 https://github.com/haskell/cabal.
 
-Ways to get the `cabal-install` binary
---------------------------------
+## Support window for releases
+
+| Latest release | 3.14 |
+|---|---|
+| Long-Time Support release | 3.12 |
```

Comment:
As I said in the past: I'm mildly against using the term LTS without having any concrete definition for it. Here would be a great place for a sorry definition btw. But I don't know what it should be, from the top of my head.

Reply:
> We intend long-term support (LTS) releases to be stable releases with ongoing support, for use by projects which can't reasonably follow a moving target of `Cabal` or `cabal-install` releases. Previously, the `GHCup` maintainer was lightly maintaining 3.6 as a stable release, but it's unreasonable to expect him to do so for new releases, so we are providing a maintained stable release ourselves. New LTS major releases will happen as needed, but as yet we have not determined when because we don't have a good feel for how often it will be needed. The LTS release will receive major bug fixes but avoid breaking changes, which will require a new LTS series.

An example would be the 3.14 release that's in progress (or just happened?), which is fairly close on the heels of 3.12.1.0. I can well imagine other projects wanting to stick to a stable but maintained release.

Reply:
It's a good start, I guess, thank you. It feels vague because LTS policies usually have exact time frames.

Reply:
That's because it _is_ vague, and states why it is vague ("as yet we have not determined…"). We weren't even sure 3.12 would be an LTS branch at release time.

Reply:
As for timeframes, we need to find out how hard ghc releases will be pushing us along before we know how long-term the long-term support branches need to be in order to be meaningful.